CHANGELOG.md

```diff
@@ -232,3 +232,4 @@ v2.2 as released by Bitly.
   - Repository forked on 27/11/18
     - README updated to include note that this repository is forked
     - CHANGLOG created to track changes to repository from original fork
+    - Added bash >= 4.0 dependency to configure script 
```

Comment:
Sorry, this should be of the format:
```
- [#292](https://github.com/pusher/oauth2_proxy/pull/292) Added bash >= 4.0 dependency to configure script (@jmfrank63)
```
And should be at the top of this file with the more recent changes (changes since v4.0.0)